content-filter/content-filter.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+
+type ReimbursementTemplate record {
+    string reimbursementTypeID;
+    string reimbursementTypeName;
+    float? fixedAmount;
+    float? ratePerUnit;
+    float? numberOfUnits;
+};
+
+type XeroRequest record {
```

Comment:
This should also be some kind of `ReimbursementTemplate`. So maybe rename other one to `DetailedReimbursementTemplate`